.github/.PULL_REQEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## General information
+
+Use this template to provide (short) bio here and please add relevant
+information down below. To be pasted in the comment to your pull request.
+
+## self assessment:
```

Comment:
Let's just have this be the top level category (`Self Assessment:`) and include a small introductory message under it like: "Please fill out the following self assessment."